Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx

```diff
@@ -50,13 +50,16 @@ export const ManageLuis = (props: ManageLuisProps) => {
     <ManageService
       createService={createService}
       handoffInstructions={formatMessage(
-        '1. Using the Azure portal, please create a Language Understanding resource on my behalf.\n2. Once provisioned, securely share the resulting credentials with me as described in the link below.\n\nDetailed instructions:\nhttps://aka.ms/bfcomposerhandoffluis'
+        '1. Using the Azure portal, please create a Language Understanding resource.\n2. Once created, securely share the resulting credentials with me as described in the link below.\n\nDetailed instructions:\nhttps://aka.ms/bfcomposerhandoffluis'
```

Comment:
this link is broken

Reply:
this link is being created by the docs team its on their radar.